src/main/groovy/life/qbic/datamodel/people/Person.groovy

```diff
@@ -1,5 +1,6 @@
 package life.qbic.datamodel.people;
 
+@Deprecated
```

Comment:
Did we want to deprecate this Person class? I think we don't even use it in the context of TOMATO.

Reply:
I think it would be a good idea to deprecate classes with the same name but different implemenations to avoid confusion in the future. I don't know in which other classes this class is used, therefore we discussed deprecating it and removing it as soon as we know how easy it is to do so